src/Parser/HolyPriest/CombatLogParser.js

```diff
@@ -214,6 +219,15 @@ class CombatLogParser extends MainCombatLogParser {
           </dfn>
         )}
       />,
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.HOLY_WORD_SANCTIFY.id} />}
+        value={`${sancAvgHits.toFixed(2)}`}
+        label={(
+          <dfn data-tip={`A measure of how many targets were effectively healed by your Holy Word: Sanctify. Over 80% overhealing on a hit is considered a "miss". You missed ${sancMissedHits} of ${this.modules.sanctify.casts * 6} potential hits.`}>
+            Average Sanctify hits
```

Comment:
This needn't say *Sanctify* as that should already be obvious from the icon, so it would become just *Average hits*. (consistent with other places)